CONTRIBUTING.md

```diff
@@ -1,189 +1,272 @@
+build:vcluster Done processing image 'ghcr.io/loft-sh/loft-enterprise/dev-vcluster'
+deploy:vcluster-k8s Deploying chart ./chart (vcluster) with helm...
+deploy:vcluster-k8s Deployed helm chart (Release revision: 1)
+deploy:vcluster-k8s Successfully deployed vcluster-k8s with helm
+dev:vcluster Waiting for pod to become ready...
+dev:vcluster DevSpace is waiting, because Pod vcluster-devspace-7b467dc9cd-hbtbz has status: Init:1/3
+dev:vcluster Selected pod vcluster-devspace-7b467dc9cd-hbtbz
+dev:vcluster ports Port forwarding started on: 2346 -> 2345
+dev:vcluster sync  Sync started on: . <-> .
+dev:vcluster sync  Waiting for initial sync to complete
+dev:vcluster sync  Initial sync completed
+dev:vcluster term  Opening shell to syncer:vcluster-devspace-7b467dc9cd-hbtbz (pod:container)
+
+   ____              ____
+  |  _ \  _____   __/ ___| _ __   __ _  ___ ___
+  | | | |/ _ \ \ / /\___ \| '_ \ / _` |/ __/ _ \
+  | |_| |  __/\ V /  ___) | |_) | (_| | (_|  __/
+  |____/ \___| \_/  |____/| .__/ \__,_|\___\___|
+                          |_|
+
+Welcome to your development container!
+This is how you can work with it:
+- Run `go run -mod vendor cmd/vcluster/main.go start` to start vcluster
+- Run `devspace enter -n vcluster --pod vcluster-devspace-7b467dc9cd-hbtbz -c syncer` to create another shell into this container
+- Run `kubectl ...` from within the container to access the vcluster if its started
+- Files will be synchronized between your local machine and this container
+
+NOTE: you may need to provide additional flags through the command line, because the flags set from the chart are ignored in the dev mode.
+
+If you wish to run vcluster in the debug mode with delve, run:
+  `dlv debug ./cmd/vcluster/main.go --listen=0.0.0.0:2345 --api-version=2 --output /tmp/__debug_bin --headless --build-flags="-mod=vendor" -- start`
+  Wait until the `API server listening at: [::]:2345` message appears
+  Start the "Debug vcluster (localhost:2346)" configuration in VSCode to connect your debugger session.
+  Note: vcluster won't start until you connect with the debugger.
+  Note: vcluster will be stopped once you detach your debugger session.
+
+TIP: hit an up arrow on your keyboard to find the commands mentioned above :)
+
+vcluster-0:vcluster-dev$ 
+```
+
+Now, your terminal is running in DevSpace, and you can develop and test with DevSpace. 
 
-#########################################################
-[info]   DevSpace UI available at: http://localhost:8090
-#########################################################
+### Developing and Testing vCluster using DevSpace 
 
-[0:sync] Waiting for pods...
-[0:sync] Starting sync...
-[0:sync] Sync started on /Users/fabiankramm/Programmieren/go-workspace/src/github.com/loft-sh/vcluster <-> . (Pod: vcluster/vcluster-0)
-[0:sync] Waiting for initial sync to complete
-[info]   Opening shell to pod:container vcluster-0:syncer
-root@vcluster-0:/vcluster#
+Start vcluster in DevSpace
```

Comment:
As we are showing the output of DevSpace above, the commands to run vcluster via `go run` or `dlv` are already mentioned there. I guess the redundancy is ok, but we might want to refer to the output above?!

```suggestion
#### Start vcluster in DevSpace via `go run`

As suggested by the DevSpace output above you can start vcluster via `go run`: 
```